test-dsl/src/main/groovy/com/smartbear/readyapi4j/dsl/assertions/AbstractAssertionsDelegate.groovy

```diff
@@ -0,0 +1,116 @@
+package com.smartbear.readyapi4j.dsl.assertions
+
+import com.smartbear.readyapi4j.assertions.AssertionBuilder
+import com.smartbear.readyapi4j.assertions.Assertions
+import com.smartbear.readyapi4j.assertions.ContainsAssertionBuilder
+import com.smartbear.readyapi4j.assertions.InvalidHttpStatusCodesAssertionBuilder
+import com.smartbear.readyapi4j.assertions.ValidHttpStatusCodesAssertionBuilder
+
+import static com.smartbear.readyapi4j.assertions.Assertions.contains
+import static com.smartbear.readyapi4j.assertions.Assertions.notContains
+
+/**
+ * An abstract class which contains common assertions applicable to multiple requests: SOAP, REST etc.
+ */
+abstract class AbstractAssertionsDelegate {
```

Comment:
I had kept this way because we allow to add all those assertions to pretty much all test steps. But I agree it will be much better to have only test step specific assertions available in DSL.